docs/schedule/index.md

```diff
@@ -28,20 +28,25 @@ The web-server hosting GTFS data should be configured to correctly report the fi
 
 See “[Best Practices: Dataset Publishing](best-practices/#dataset-publishing-general-practices)” for further recommendations.
 
-## Training
+## Training & Resources
 <hr>
 
-The World Bank Open Learning Campus (OLC) offers a self-based online course called “[Introduction to the General Transit Feed Specification (GTFS) and Informal Transit System Mapping](https://olc.worldbank.org/content/introduction-general-transit-feed-specification-gtfs-and-informal-transit-system-mapping)”. This course includes the following sections:
-
-- What is GTFS? History & File Structure
-- What is GTFS? Visualization & Community
-- Setting up a GTFS Feed
-- Introduction to GitHub & Open Source Tools
-- Stories from the Field
-- How to Map Transit Data
-- How to Collect Data for a City’s First Feed
-- App Survey
-- GTFS-Realtime
+**Technical details about GTFS, what it is, and how to create and maintain data:**
```

Comment:
It's not totally clear to me why from the markdown, but it looks like the list formatting is breaking when I test this page locally: 

<img width="1148" alt="Screenshot 2023-05-16 at 5 21 23 PM" src="https://github.com/MobilityData/gtfs.org/assets/90632321/970d0793-3880-4a3f-9e71-b552fcaf0265">

Reply:
This *might* be due to the differences between the [*CommonMark*](https://commonmark.org) (colloquially the "standard markdown") and [GitHub-flavored Markdown](https://github.github.com/gfm/), specifically about the different treatment of single vs double line breaks after headings...